modules/libcom/src/timer/timerPrivate.h

```diff
@@ -35,6 +35,15 @@
 #   define debugPrintf(ARGSINPAREN)
 #endif
 
+/* For historical reasons, round down expiration time on vxWorks and RTEMS.
+ * This allows timers which expire on every tick.
+ * Otherwise, the fastest timer period is 2x the tick interval.
+ * This results in timers expiring early.
```

Comment:
It's unclear to me what "This" in line 41 is actually referring to; for the sanity of future maintainers it would be good to clarify that wording.
Also "For historical reasons" could become "On OSs with lo-res timers".